exonum-java-binding/qa-service/src/main/java/com/exonum/binding/qaservice/QaService.java

```diff
@@ -50,34 +44,10 @@
 
   Optional<Counter> getValue(HashCode counterId);
 
-  Height getHeight();
-
-  List<HashCode> getBlockHashes();
-
-  List<HashCode> getBlockTransactions(long height);
-
-  List<HashCode> getBlockTransactions(HashCode blockId);
-
-  Map<HashCode, TransactionMessage> getTxMessages();
-
-  Map<HashCode, TransactionResult> getTxResults();
-
-  Optional<TransactionResult> getTxResult(HashCode messageHash);
-
-  Map<HashCode, TransactionLocation> getTxLocations();
-
-  Optional<TransactionLocation> getTxLocation(HashCode messageHash);
-
-  Map<HashCode, Block> getBlocks();
-
-  Block getBlockByHeight(long height);
-
-  Optional<Block> getBlockById(HashCode blockId);
-
-  Block getLastBlock();
-
-  StoredConfiguration getActualConfiguration();
+  // todo: Is configuration still needed?
+  // StoredConfiguration getActualConfiguration();
```

Comment:
@dip56 could you please check that your tests don't check anything extra compared to `BlockchainIntegrationTest#getActualConfiguration`?